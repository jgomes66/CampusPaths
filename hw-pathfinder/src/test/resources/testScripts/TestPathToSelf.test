# Simple test to see what happens when find path
# is used and the same building is both the start and end

CreateGraph simpleGraph

AddNode simpleGraph v1

FindPath simpleGraph v1 v1

