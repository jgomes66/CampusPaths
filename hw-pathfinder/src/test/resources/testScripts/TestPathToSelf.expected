# Simple test to see what happens when find path
# is used and the same building is both the start and end

created graph simpleGraph

added node v1 to simpleGraph

path from v1 to v1:
total cost: 0.000